### DISCUSSION TITLE
How to export data into Excel from Jamku

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#608] [QA] [answered]
What are the ways in which Data can be exported to excel from Jamku?

**Comment by Divwork92** [comment, marked as answer]
You can export all data from Jamku into Excel. There are predominantly 2 ways of exporting:

1. Click on the excel icon, which you can find on the right side of the page and export OR
2. Switch to table view and right click -> export -> excel export
